modules/rostests/kmtests/ntos_ob/ObQuery.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * PROJECT:     ReactOS kernel-mode tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Kernel mode tests for object information querying
+ * COPYRIGHT:   Copyright 2023 George Bișoc <george.bisoc@reactos.org>
+ */
+
+#include <kmt_test.h>
+
+#define OBJ_WINSTA_DIRECTORY_NAME_INFO_SIZE sizeof(UNICODE_STRING) + sizeof(L"\\Windows")
+#define OBJ_WINSTA_DIRECTORY_TYPE_INFO_SIZE sizeof(OBJECT_TYPE_INFORMATION) + sizeof(L"\\Windows") + sizeof(UNICODE_NULL)
```

Comment:
Shouldn't this one be based on the string "Directory", not the string "\Windows"?